homeassistant/components/nest/media_source.py

```diff
@@ -56,6 +60,177 @@
 CLIP_TITLE_FORMAT = "{event_name} @ {event_time}"
 EVENT_MEDIA_API_URL_FORMAT = "/api/nest/event_media/{device_id}/{event_id}"
 
+STORAGE_KEY = "nest.event_media"
+STORAGE_VERSION = 1
+# Write events at most once per duration
+STORAGE_SAVE_DELAY_SECONDS = 5.0
```

Comment:
Why not make this 60 seconds or even longer ? 

It's guaranteed to be written at shutdown.

Reply:
Ah, i was worried about shutdown and inconsistencies between the metadata store and files.  Great, increased to 2 minutes.